src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -142,7 +142,8 @@
 								:conversations="filteredConversationsList"
 								:loading="!initialisedConversations"
 								class="scroller h-100"
-								@scroll.native="debounceHandleScroll" />
+								@scroll.native="debounceHandleScroll"
+								@item-clicked="abortSearch" />
```

Comment:
I'd listen instead of `<Conversation @click="abortSearch" />` here:

LeftSidebar => `onRouteChange()` =>  `if (to.name === 'conversation')` => `this.abortSearch()`